### DISCUSSION TITLE
A Serverless Function has exceeded the unzipped maximum size of 250 MB. - But i dont use serverless functions!

### DISCUSSION THREAD

**Discussion by enormousrodent** [vercel/community#4423] [Help]
### Summary

Hi,  I have a gatsby site based on a repo that depending on the configuration can be deployed in different configurations (i.e. its a global travel / hotel site and i have regional sites (e.g. Phuket Thailand) and then i have the 'main' site which is Global.  The only difference on deployment is the number of pages that get deployed.  The Regional sites are only 2-300 pages where as the global site has 15k.

Anyhow, these sites build perfectly locally and when the global site had viewer pages it also deployed fine.  I had a problem before that my site was not very well optimised so i have spent the last couple of days getting the payloads and total site down by 80% which means my large 15k site is building in less than 20 minutes.  

Now the problem is that i am getting an issue saying :

A Serverless Function has exceeded the unzipped maximum size of 250 MB.

The issue is, i am not using any vercel serverless functions in my project so i don't know what this 250mb is referring to.  The logs do not show any specific issues relating to this (last few lines of logs shown below:)

19:22:56.831 | │                                                                    │
19:22:56.831 | │     (SSG) Generated at build time                                  │
19:22:56.832 | │   D (DSG) Deferred static generation - page generated at runtime   │
19:22:56.832 | │   ∞ (SSR) Server-side renders at runtime (uses getServerData)      │
19:22:56.832 | │   λ (Function) Gatsby function                                     │
19:22:56.832 | │                                                                    │
19:22:56.832 | ╰────────────────────────────────────────────────────────────────────╯
19:22:56.832 | info Done building in 1378.68202376 sec
19:22:58.005 | Build Completed in /vercel/output [23m]
19:23:14.811 | Deploying outputs...
19:24:54.484 | Error: A Serverless Function has exceeded the unzipped maximum size of 250 MB. : https://vercel.link/serverless-function-size

So as you can see the site is actually 'built' fine and i cannot see any single file in the build output (when locally built) that is >250 MB.  I also do not use any kind of image processing as all my images are hosted externally via CDN.

If anyone has any idea how i can diagnose this issue that would be greatly appriciated..

Thanks

Chris

### Steps to Reproduce

Build site via vercel...wait...cry :(

**Comment by rickylabs** [comment]
Same issue here site deploys without issue on Gatsby Cloud

**Comment by amyegan** [comment]
Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that I can use to get more information about the problem?
